src/App.js

```diff
@@ -4,45 +4,596 @@
 import React from "react";
 // DOM bindings for React Router
 import { BrowserRouter as Router } from "react-router-dom";
+//> Additional
+// SHA Hashing algorithm
+import sha256 from "js-sha256";
+//> Intel
+import { Intel } from "snek-intel";
+//> MDB
+// "Material Design for Bootstrap" is a great UI design framework
+import { MDBProgress } from "mdbreact";
 
 //> Components
 /**
  * Footer: Global Footer
  * Navbar: Global navigation bar
  */
-//import { Footer, Navbar } from "./components/molecules";
-
+import { Footer, Navbar } from "./components/molecules";
+// Starts the page on top when reloaded or redirected
+import { ScrollToTop } from "./components/atoms";
 //> Routes
 import Routes from "./Routes";
 //#endregion
 
 //#region > Components
+/**
+ * @class Root component which loads all other components
+ */
```

Comment:
Has to be discussed in a meeting, for now I will leave this request open as a reminder.

Reply:
> Has to be discussed in a meeting, for now I will leave this request open as a reminder.

Please reference this topic in a new issue. This can be done directly in this comment.

See:
![image](https://user-images.githubusercontent.com/52858351/84406685-2c3bfa00-ac0a-11ea-8f52-95398b68f8d6.png)

Reply:
It sould be discussed, if a folder for `helpers` should be made which would contain most of those functions - only to be imported into the `App.js` to be appended to globalFunctions.

Reply:
> It sould be discussed, if a folder for `helpers` should be made which would contain most of those functions - only to be imported into the `App.js` to be appended to `globalFunctions`.

I think this is a good way to go. But is this relevant to v0.1.0? @snek-at/frontend

Reply:
I would certainly appreciate this being part of a very early version to keep building on that solid base, yes.

Reply:
> > It sould be discussed, if a folder for `helpers` should be made which would contain most of those functions - only to be imported into the `App.js` to be appended to `globalFunctions`.
> 
>> I think this is a good way to go. But is this relevant to v0.1.0? @snek-at/frontend 
>>> I would certainly appreciate this being part of a very early version to keep building on that solid base, yes.

After merging https://github.com/snek-at/front/pull/63 I will add it.